pkgs/development/python-modules/notebook/default.nix

```diff
@@ -44,8 +44,8 @@ buildPythonPackage rec {
   # disable warning_filters
   preCheck = lib.optionalString (!isPy3k) ''
     echo "" > setup.cfg
```

Comment:
This could be replaced by `touch`

Reply:
Good point, updated.

Reply:
It just occurred to me that this was probably intentional to truncate the file. Should we reinstate it?

Reply:
Yup, it seems to fail when building with Python 2. I've opened #37620.